_config.yml

```diff
@@ -2,7 +2,7 @@ presto_version: 340
 
 google_analytics: UA-133457846-1
 
-slack_url: "https://join.slack.com/t/prestosql/shared_invite/enQtNTMyNzU2NzQ1NzQ4LWUyMjY4NTA0NWU0ZDlkMjM4Y2JhODdiZWZiZWQ2NDNhZDVhOGY5ZTk4NDY0OGIzOTE2YzI3YjhlMmUyNDJiNGM"
+slack_url: "https://join.slack.com/t/prestosql/shared_invite/zt-h88upx5f-gVb8XUu16W_jo4I2Cw8ijA"
```

Comment:
Why this change?

Reply:
I'm not sure why this is showing in the diff. This is the line that is currently on master. https://github.com/prestosql/prestosql.io/blob/master/_config.yml#L5
This change was merged in from master but shouldn't be showing as one of the files changed.

Reply:
> This change was merged in from master but shouldn't be showing as one of the files changed.

oh, perhaps because you used `git merge`. We do not merge, please `git rebase -i master` instead

Reply:
Just needed to merge in upstream master.

Reply:
okay, removed the extra commit and squashed some extra changes Martin asked me to make.